0001-chore-set-boto-minimum-versions.patch

```diff
@@ -0,0 +1,32 @@
+From fd98187a80c3e914ffcdc28e73c0dbb29ccf12fb Mon Sep 17 00:00:00 2001
+From: Armando Neto <abiagion@redhat.com>
+Date: Mon, 8 Feb 2021 17:59:15 -0300
+Subject: [PATCH] chore: set boto minimum versions
+
+Setting the minimal version of all packages instead of pinning their
+exact version, in preparation of packaging mrack as RPM.
+
+Setting the minimium version to the ones availble in the stable repos of
+Fedora 32.
+
+Signed-off-by: Armando Neto <abiagion@redhat.com>
+---
+ requirements.txt | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/requirements.txt b/requirements.txt
+index ab2705f..9521dbe 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -1,6 +1,6 @@
+ beaker-client>=28
+-botocore==1.18.18
+-boto3==1.15.18
++botocore>=1.14.17
```

Comment:
The patch will not be applicable once we release a new version with the amended requirements.txt.